deploy/crds/logging.openshift.io_collectors_crd.yaml

```diff
@@ -9,7 +9,7 @@ spec:
     listKind: CollectorList
     plural: collectors
     singular: collector
-  scope: Namespaced
+  scope: ""
```

Comment:
indeed it was due to not using latest `operator-sdk`. deleted and downloaded again `v0.16.0`

Reply:
Should we fix to use a version? otherwise modify script to check for a newer version and download it. now it only downloads latest if no `operator-sdk` is present.

Reply:
The actually fix is to update CLO to use go-modules and then gradually operator-sdk v0.16.0 as in openshift/elasticsearch-operator/pull/291. Finally the Makefile calls to operator-sdk should be of the sort `go run ./vendor/operator-sdk...` instead of downloading anything. I think we are d'accord here with @alanconway